frontend/src/__tests__/cypress/cypress/tests/mocked/projects/clusterStorage.cy.ts

```diff
@@ -109,7 +109,12 @@ describe('ClusterStorage', () => {
       initInterceptors({ storageClassName: 'openshift-default-sc' });
       clusterStorage.visit('test-project');
       const clusterStorageRow = clusterStorage.getClusterStorageRow('Test Storage');
-      clusterStorageRow.findStorageClassColumn().should('exist');
+      const storageClass = clusterStorageRow.findStorageClassColumn();
+      storageClass.should('exist');
+      storageClass.should('have.text', 'openshift-default-sc');
+      storageClass.findByTestId('resource-name-icon-button').click();
+      cy.findByTestId('resource-name-text').should('have.text', 'openshift-default-sc');
+      cy.findByTestId('resource-kind-text').should('have.text', 'StorageClass');
```

Comment:
can you create a function to these and move it to `frontend/src/__tests__/cypress/cypress/pages/clusterStorage.ts`